tests/phpunit/tests/kses.php

```diff
@@ -1353,6 +1354,16 @@ public function data_safecss_filter_attr() {
 				'css'      => 'background-repeat: no-repeat',
 				'expected' => 'background-repeat: no-repeat',
 			),
+			// Convert "&amp;" to "&".
+			array(
+				'css'      => 'background: url("/sites/2/2023/10/image.jpg?width=1024&amp;height=600")',
+				'expected' => 'background: url("/sites/2/2023/10/image.jpg?width=1024&height=600")',
+			),
+			// Continues to handle "&".
```

Comment:
Added a dataset to confirm it continues to handle "&", following @peterwilsoncc [good suggestion](https://github.com/WordPress/wordpress-develop/pull/6645/files#r1616422804).